framework/components/AButtonCard/AButtonCard.scss

```diff
@@ -1,91 +1,100 @@
 @import "../../styles";
 
-// -----------------------------------------------------------------------------
-// Button Card Default
-// -----------------------------------------------------------------------------
 $btn-card-min-width: 350px;
 $btn-card-min-height: 150px;
 $btn-card-padding: 20px;
 $btn-card-title-margin-bottom: 10px;
 $btn-card-font-size: $font-size--md;
 $btn-card-bar-height: -5px;
```

Comment:
Can you please distribute these values and remove the sass variables? They're not set up to be overridden and we don't have guidance on what should be overridable